src/Microsoft.AspNetCore.Mvc.Abstractions/project.json

```diff
@@ -33,17 +33,23 @@
       "version": "1.1.0-*",
       "type": "build"
     },
+    "Microsoft.Extensions.Localization.Abstractions": "1.1.0-*",
```

Comment:
This new dependency may have implications w.r.t. binary compatibility when using just `Microsoft.AspNetCore.Mvc.Core`. Thoughts @rynowak?